web-app/src/components/ServicesPage/index.js

```diff
@@ -2,60 +2,51 @@ import React, { useState, useEffect } from "react";
 
 import { Box, Container } from "@material-ui/core";
 import { ToggleButton, ToggleButtonGroup } from "@material-ui/lab";
+import { GoogleApiWrapper, Map, Marker } from "google-maps-react"
 
 import Service from "./Service";
+import { myServices } from "../../constants/services"
 
-const ServicesPage = () => {
-  //initialstate
-  const myServices = [
-    {
-      title: "Provider 1",
-      address: "123 Main St",
-      services: ["Food", "Jobs"],
-    },
-    {
-      title: "Provider 2",
-      address: "411 E Wisconsin Ave",
-      unit: "9th Fl",
-      zip: "53202",
-    },
-    {
-      title: "Provider 3",
-      address: "808 N 3rd St",
-      city: "Milwaukee",
-      state: "WI",
-      zip: "53203",
-    },
-  ];
-
-  const [expanded, setExpanded] = useState("service0");
+const ServicesPage = (props) => {
+  const [expanded, setExpanded] = useState(1);
   const [segments, setSegments] = useState([]);
-  const [currentServiceExpanded, setCurrentService] = useState(
-    "No service selected"
-  );
```

Comment:
I took this out because I thought we'd replace the name of the selected service with a pin on the map - let me know if you want me to put it back!

Reply:
I like this change, this makes sense. I'm recruiting lived experience testers right now so we'll put it to the test instead of going off preference (unless anyone has UX experience)